war/src/main/less/base/yui-compatibility.less

```diff
@@ -95,6 +95,14 @@
   color: inherit;
 }
 
+.ygtvfocus {
+  background-color: var(--panel-header-bg-color, #c0e0e0) !important;
```

Comment:
Not a blocker, but could you try using on the greys used on the palette? You can look at that on the theme.less file.

Reply:
I picked `--panel-header-bg-color` because I think that every theme will set that to something that contrasts with the background colour.
Did you have a specific colour in mind?

Reply:
I meant for the fallback but now that I think of it it's not needed

Reply:
Yeah I just kept the fallback the old value.